client/src/components/Slider/Slider.stories.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+
+import Slider from "./Slider";
+
+import imageFile1 from "../../Images/Fall-2020-1.png";
+import imageFile2 from "../../Images/Fall-2020-2.png";
+import imageFile3 from "../../Images/Spring-2020-1.jpeg";
+import imageFile4 from "../../Images/Spring-2020-2.jpeg";
+import imageFile5 from "../../Images/Trending-Now-1.jpeg";
+import imageFile6 from "../../Images/Trending-Now-2.jpeg";
+import imageFile7 from "../../Images/Classics-1.jpeg";
+import imageFile8 from "../../Images/Classics-2.jpeg";
+
+const slideData = [
+  {
+    title: "Fall 2020 Collection",
+    imga: imageFile1,
+    imgb: imageFile2,
+    labela: "LUCA",
+    labelb: "VIDA",
+    index: 0
+  },
+  {
+    title: "Fall 2020 Collection",
+    imga: imageFile3,
+    imgb: imageFile4,
+    labela: "AlWAYS",
+    labelb: "BELIZE",
+    index: 1
+  },
+  {
+    title: "Fall 2020 Collection",
+    imga: imageFile5,
+    imgb: imageFile6,
+    labela: "ALICE",
+    labelb: "EVELYN",
+    index: 2
+  },
+  {
+    title: "Fall 2020 Collection",
+    imga: imageFile7,
+    imgb: imageFile8,
+    labela: "AINSLEY",
+    labelb: "ANNIE",
+    index: 3
+  }
+];
+
+storiesOf("Slider", module).add("Gown Display Slider", () => (
+  <Slider slideData={slideData} title="Classics"></Slider>
```

Comment:
How is the title being used here?

Reply:
That is an excellent question...I think it was in a mock up elsewhere and I added it here when we originally started this assignment.  I have removed it as it can be added in the molecule this build goes into.